src/store/store.js

```diff
@@ -262,11 +262,27 @@ export default new Vuex.Store({
 			}
 			return true
 		},
+		async updateTableProperty({ state, commit, dispatch }, { id, data, property }) {
```

Comment:
I can't find where this function is being used? Maybe it is no longer needed?

Reply:
You're right that's an artifact from when the description was editable in the tables view.